tests/Fixer/Phpdoc/PhpdocAnnotationWithoutDotFixerTest.php

```diff
@@ -139,6 +139,22 @@ public function provideFixCases()
      *                   and Returns `false` otherwise.
      */',
             ],
+            [
+                '<?php
+    /**
+     * This is a broken phpdoc
+     * @param string string Surprisingly, it is a string
+
+     */
+    function fixMe($str) {}',
+                '<?php
+    /**
+     * This is a broken phpdoc
+     * @param string string Surprisingly, it is a string.
```

Comment:
We can see that the fixer works by the removal of the dot, but this should be `string $str` instead of `string string`, and then `Surprisingly` will be lowercased as well.